test/timeout.js

```diff
@@ -10,26 +10,44 @@ test.before('setup', async () => {
 	s = await createServer();
 
 	s.on('/', async (req, res) => {
-		await delay(20);
+		await delay(80);
 		res.statusCode = 200;
 		res.end('OK');
 	});
 
 	await s.listen(s.port);
 });
 
-test('timeout option', async t => {
-	const err = await t.throws(got(`${s.url}/`, {
-		timeout: 1,
+test('timeout option (ETIMEDOUT)', async t => {
+	const err = await t.throws(got(`${s.url}`, {
+		timeout: 0,
+		retries: 0
+	}));
+
+	t.is(err.code, 'ETIMEDOUT');
```

Comment:
For new assertions, we should use the improve `t.throws` API: https://github.com/avajs/ava#throwsthrower-expected-message

In this case, it would be:

```js
await t.throws(
	got(s.url, {
		timeout: 0,
		retries: 0
	}),
	{
		code: 'ETIMEDOUT'
	}
);
```